src/Items/ItemFishingRod.h

```diff
@@ -234,6 +234,10 @@ class cItemFishingRodHandler :
 				a_Player->UseEquippedItem(1);
 				cRoot::Get()->GetPluginManager()->CallHookPlayerFished(*a_Player, Drops);
 			}
+			else if (!IsBlockWater(a_World->GetBlock(FloaterInfo.GetPos())))
```

Comment:
This doesn't seem right.  If the floater is in the air that's still not water but it shouldn't be taking damage.  I think `IsSolid` would be better, or whatever the floater's physics uses for collisions would probably be best.

Reply:
Thanks, I didn't think about that.